### PR TITLE
API fixes and add links

### PR DESCRIPTION
**Pull request by dqii**

### CODE REVIEW

File: lanterndb/lantern.ipynb [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "api_url = \"https://api.ecliptor.com/ingest/pdf\"\n",
     "# Request payload\n",
     "payload = {\n",
-    "    \"url\": pdf_url\n",
+    "    \"url\": api_url\n",
```
Comment (nankigrewal) [inline lanterndb/lantern.ipynb]: This is actually gonna be the pdf_url - the post request is to the API endpoint but what we need is a URL that leads to the pdf

File: lanterndb/lantern.ipynb [hunk 2]
```diff
@@ -249,38 +254,14 @@
...
-    "embedded_queries = [embed_text(query) for query in query_list]"
+    "### Create queries from each of the texts into the index"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, perform lookups within the vector database using the embedded queries."
+    "Now we will perform lookups within the vector database using the embedded queries, using same embeddings model used to embed the entire corpus."
```
Comment (nankigrewal) [inline lanterndb/lantern.ipynb]: nit: add "the" to "using same embeddings model..."

File: lanterndb/lantern.ipynb [hunk 3]
```diff
@@ -161,16 +161,21 @@
...
+    "cur.execute(\"\"\"\n",
+    "\tALTER DATABASE postgres SET lantern_extras.openai_token='OPENAI_KEY';\n",
+    "\"\"\")\n",
+    "cur.execute(\"\"\"\n",
     "\tSELECT add_embedding_job(\n",
     "\t    'compliance_documents',         -- Name of the table\n",
     "\t    'chunk',                        -- Source column for embeddings\n",
     "\t    'vector',                       -- Destination column for embeddings\n",
-    "\t    'openai/text-embedding-3-large' -- Embedding model to use\n",
+    "\t    'openai/text-embedding-3-small' -- Embedding model to use\n",
```
Comment (nankigrewal) [inline lanterndb/lantern.ipynb:180]: any reason we'd prefer smaller dimensions for this blog?

**Review by nankigrewal — APPROVED** [review]

File: lanterndb/lantern.ipynb (hunk 3 above)
Comment (dqii) [reply lanterndb/lantern.ipynb:180]: text-embedding-3-small works better with Postgres and we haven't seen notably different performance for the additional cost

**Pull request merged**